src/util/upnp_headers.cc

```diff
@@ -25,7 +25,7 @@
 
 #include "upnp_headers.h" // API
 
-#if (UPNP_VERSION > 11201)
+#if (UPNP_VERSION_MINOR > 12)
```

Comment:
In order to be really precise you  also need to add `UPNP_VERSION_MAJOR > 1 ||`
Anyway, we're discussing things that do not exist!

Reply:
Not quite. My installation of libupnp has a version of 12.2 (some old git revision).

version major won't get bumped. Based on how fast Gerbera moves, I assume this whole block will go away.

Reply:
Interesting, officially PUPnP goes from 1.12.1 to 1.14.0. Best suggestion is `UPNP_VERSION > 11299` then.

Reply:
Can we add the major version too? Just avoid any chance of weirdness

Reply:
Went with Karl's suggestion. This define will probably be removed at some point.